dist/PublicLab.Editor.js

```diff
@@ -22475,7 +22475,7 @@ module.exports = function(textarea, _editor, _module) {
     );
     $(".wk-commands, .wk-switchboard").addClass("btn-group");
     $(".wk-commands button, .wk-switchboard button").addClass(
-        "btn btn-light"
+        "btn btn-outline-secondary"
```

Comment:
@NARUDESIGNS What is this change for?

Reply:
@TildaDares the change keeps happening automatically every time I make changes to src/modules/PublicLab.RichTextModule.Table.js

Reply:
I've left a comment that explains this in #757

Reply:
Sorry, I'm just responding now. I followed your instruction in #757 and I've made all necessary changes as you required.
@TildaDares